tensorrt_yolov9_ros:
    image_compressed: false                 # true: use compressed image, false: use raw image
    image_topic: "/camera/image_raw"        # image topic
    engine_file_path: "/home/mason/ws/yolo_ws/yolov9/yolov9-c_fp16.engine"
    confidence_thres: 0.5                   # confidence threshold
    nms_thres: 0.4                          # nms threshold
    downsampling_infer: 1                   # inference every n frames, if 3, then 1/3 of images
    classes: ["person", "bicycle", "car", "motorbike", "aeroplane", "bus",
              "train", "truck", "boat", "traffic light", "fire hydrant",
              "stop sign", "parking meter", "bench", "bird", "cat", "dog",
              "horse", "sheep", "cow", "elephant", "bear", "zebra", "giraffe",
              "backpack", "umbrella", "handbag", "tie", "suitcase", "frisbee",
              "skis", "snowboard", "sports ball", "kite", "baseball bat",
              "baseball glove", "skateboard", "surfboard", "tennis racket",
              "bottle", "wine glass", "cup", "fork", "knife", "spoon", "bowl",
              "banana", "apple", "sandwich", "orange", "broccoli", "carrot",
              "hot dog", "pizza", "donut", "cake", "chair", "sofa", "pottedplant",
              "bed", "diningtable", "toilet", "tvmonitor", "laptop", "mouse",
              "remote", "keyboard", "cell phone", "microwave", "oven", "toaster",
              "sink", "refrigerator", "book", "clock", "vase", "scissors", "teddy bear",
              "hair drier", "toothbrush"]